torchvision/transforms.py

```diff
@@ -39,38 +39,60 @@ def __call__(self, pic):
         if isinstance(pic, np.ndarray):
             # handle numpy array
             img = torch.from_numpy(pic.transpose((2, 0, 1)))
+            # backard compability
+            return img.float().div(255)
+        # handle PIL Image
+        if pic.mode == 'I':
+            img = torch.from_numpy(np.array(pic, np.int32))
```

Comment:
Could you add the option `copy=False` into the constructor of `np.array`? This way we avoid an unnecessary memory copy (which will slow down things a bit)